### PR TITLE
[New] Manually Enable Floating IP on your Linode

### PR DESCRIPTION
**Pull request by leslitagordita**
* This is a hiddenguide. Only a few Linode users will need to access the steps contained in this guide.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit cd2dbfdf7998c37031bafba85b3cf4580fcc91bf

https://deploy-preview-3395--nostalgic-ptolemy-b01ab8.netlify.app

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit d1fa9ac3ac9d7ca5710c1811f7bbcb3625b17f9c

https://deploy-preview-3395--nostalgic-ptolemy-b01ab8.netlify.app

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit c2ed764fe591b5e90af3a4cbe9917417b7493364

https://deploy-preview-3395--nostalgic-ptolemy-b01ab8.netlify.app

**Review by rsyracuse — APPROVED** [review]

File: docs/platform/manager/manually-enable-elastic-ip-on-your-linode/index.md [hunk 1]
```diff
@@ -0,0 +1,524 @@
...
+
+IPADDR2=[ELASTIC_IP]_2
+PREFIX2="24"
+    {{</ file >}}
+
+1. Apply the `eth0` network interface configuration:
+
+    > **Debian 10, Ubuntu 18.04 & CentOS 8**
+    >
+    > Edit your Linode's `/etc/network/interfaces` file with the following entries. Replace `[ELASTIC_IP]` with the Elastic IPv4 address:
```
Comment (andystevensname) [inline docs/platform/manager/manually-enable-elastic-ip-on-your-linode/index.md:485]: Is there meant to be an instance of `[ELASTIC_IP]` in this example?

File: docs/platform/manager/manually-enable-elastic-ip-on-your-linode/index.md [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+    >
+    >
+        sudo ifdown eth0
+        sudo ifup eth0
+    > **Ubuntu 20.04**
+    >
+    > Edit your Linode's `/etc/network/interfaces` file with the following entries. Replace `[ELASTIC_IP]` with the Elastic IPv4 address:
```
Comment (andystevensname) [inline docs/platform/manager/manually-enable-elastic-ip-on-your-linode/index.md:492]: Same as above, what am I replacing here?

**Review by andystevensname — CHANGES_REQUESTED** [review]

File: docs/platform/manager/manually-enable-elastic-ip-on-your-linode/index.md (hunk 1 above)
Comment (leslitagordita) [reply docs/platform/manager/manually-enable-elastic-ip-on-your-linode/index.md]: Removed that line.

File: docs/platform/manager/manually-enable-elastic-ip-on-your-linode/index.md (hunk 2 above)
Comment (leslitagordita) [reply docs/platform/manager/manually-enable-elastic-ip-on-your-linode/index.md]: Removed that line. Should be good now.

**Review by andystevensname — APPROVED** [review]

**Pull request merged**